src/main/java/executor/service/service/di/Application.java

```diff
@@ -0,0 +1,14 @@
+package executor.service.service.di;
+
+import java.util.Map;
+
+public class Application {
+
+    public static ApplicationContext run(String packageToScan, Map<Class, Class> ifc2ImplClass) {
+        JavaConfig config = new JavaConfig(packageToScan, ifc2ImplClass);
+        ApplicationContext context = new ApplicationContext(config);
+        ObjectFactory objectFactory = new ObjectFactory(context);
+        context.setFactory(objectFactory);
```

Comment:
ObjectFactory depends on context and context depends on ObjectFactory.
It's ok?